lib/decidim/newsletter_agenda/version.rb

```diff
@@ -2,8 +2,8 @@
 
 module Decidim
   module NewsletterAgenda
-    VERSION = "1.0.0"
-    DECIDIM_VERSION = "0.26.5"
-    COMPAT_DECIDIM_VERSION = [">= 0.26.0", "< 0.27"].freeze
+    VERSION = "1.1.0"
+    DECIDIM_VERSION = "0.27.3"
```

Comment:
```suggestion
    DECIDIM_VERSION = "0.27.4"
```
27.4 is out, we can change this (do `bundle` too)